### DISCUSSION TITLE
Custom icons?

### DISCUSSION THREAD

**Discussion by kunkel321** [freeplane/freeplane#1900] [Help]
Hello All, I'm just now coming back to Freeplane after being away for several years.  The latest version is amazing!   I'm trying to remember how to use custom home-made icons though...  Is it still possible?

**Comment by macmarrum** [comment]
Welcome back!

You can put your icon files (like svg or png) in `<user-dir>/icons` and restart Freeplane. 

Your icons will be identified by their file names. If you enabled `Preferences…->Appearance->Icons->Structured icon toolbar`, the icons will be part of `User icons`. In any case, you can search for your icon by its name:
- `Insert->Icons->Icon from table…`
- `Help->Command Search` ([x] Icons)

_**Note**: `<user-dir>` can be located by `Tools->Open user directory`_

**Reply by kunkel321** [reply]
Yes, that's right!  Thank you!

**Reply by rubcap** [reply]
I assume these need to be .ico files? I tried with .svg but these weren't recognized.

**Reply by kunkel321** [reply]
I used .png files, and it worked.

**Reply by dpolivaev** [reply]
svg should work
